src/plot_api/helpers.js

```diff
@@ -484,3 +484,19 @@ exports.manageArrayContainers = function(np, newVal, undoit) {
         np.set(newVal);
     }
 };
+
+var ATTR_TAIL_RE = /(\.[^\[\]\.]+|\[[^\[\]\.]+\])$/;
```

Comment:
Could you add maybe a single line or two about the sort of pattern this matches? I'm guessing… dot… more than one of… something… that's not brackets… just something that's not brackets or a dot? So

```
.something
something
```

match while

```
.something[]
something[]
```

do not?